adapters/api/get-downstream-assets.js

```diff
@@ -122,6 +184,180 @@ ${getImageURL(
       };
     });
   if (response.error) return response;
+  
+  const modifiedEntities = response.entities.filter(item => item.guid !== guid)
+
+  return {...response, entities: modifiedEntities}
+}
+
+function contructCommentForDownstreamLineageFetchError({
+  asset,
+  utmSource
+}){
+  const comment = `
+  ### ${getConnectorImage(asset.attributes.connectorName
+  )} [${asset.displayText}](${ATLAN_INSTANCE_URL}/assets/${
+    asset.guid
+  }/overview?utm_source=${utmSource}) ${
+    asset.attributes?.certificateStatus
+      ? getCertificationImage(asset.attributes.certificateStatus)
+      : ""
+  }
+          
+  _Failed to fetch impacted assets._
+              
+  ${getImageURL(
+    "atlan-logo",
+    15,
+    15
+  )} [View lineage in Atlan](${ATLAN_INSTANCE_URL}/assets/${
+        asset.guid
+    }/lineage/overview?utm_source=${utmSource})
+  `;
+
+  return comment;
+}
+
+export async function getDownstreamLineageForAssets({
+  asset,
+  guid,
+  totalModifiedFiles,
+  utmSource
+}) {
+  var myHeaders = {
+    authorization: `Bearer ${ATLAN_API_TOKEN}`,
+    "content-type": "application/json",
+  };
+
+  var raw = stringify({
```

Comment:
Can't we reutilize the same string that we defined above as well?

Reply:
@bichitra95 This is `/lineage/list` API payload. In entity filters we are filtering for active assets that are not of type as listed below (such as DbtProcess, DbtColumnProcess etc). This is the same payload that frontend uses to display lineage in overview panel.